.ci.yaml

```diff
@@ -3968,6 +3968,20 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Windows platform_channel_sample_test_windows
```

Comment:
@keyonghan Does this look right? As usual with .ci.yaml changes I don't do it often enough to remember how to manually test this in advance, and since it's bringup it won't be tested here.

Reply:
The ci config looks good to me.
You may want to add the test ownership entry in https://github.com/flutter/flutter/blob/master/TESTOWNERS#L218 by following:
```
# Windows platform_channel_sample_test_windows
<path-to-test> @<owner> @<team>
```

Reply:
Done. I gave it to @cbracken since desktop seemed like the best fit for it, but we could adjust it to me+plugins later if it turns out to be problematic.

Reply:
That sgtm.